.drone.yml

```diff
@@ -0,0 +1,51 @@
+kind: pipeline
+name: default
+
+platform:
+    os: linux
+    arch: amd64
+
+steps:
+- name: edelweiss_swc
+  image: mythx.azurecr.io/tools/edelweiss
+  environment:
+    IGNORE_FALSE_POSITIVES:
+      from_secret: IGNORE_FALSE_POSITIVES
+    IGNORE_REGRESSIONS:
+      from_secret: IGNORE_REGRESSIONS
+    AWS_DEFAULT_REGION:
+        from_secret: AWS_DEFAULT_REGION
+    AWS_ACCESS_KEY_ID:
+        from_secret: AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY:
+        from_secret: AWS_SECRET_ACCESS_KEY
+    MONGO_URL:
+        from_secret: MONGO_URL
+  commands:
+    - export CIRCLE_BUILD_NUM=$DRONE_BUILD_NUMBER
```

Comment:
@p1gmale0n could you add a comment to quickly explain the reason for these exports?

Reply:
added comments to job commands